### PR TITLE
Mosa 130/sync chain data

### PR DESCRIPTION
**Pull request by 0xsarvesh**
1. Sync service to sync from remote or local geth chain data to remote or local machine. 
2. Unit test cases

#145

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "edit-json-file": "1.0.8",
     "node-cmd": "3.0.0",
     "readline": "1.3.0",
+    "rsync": "^0.6.1",
     "shell-source": "1.1.0",
```
Comment (abhayks1) [inline package.json]: pls freeze version

**Review by abhayks1 — COMMENTED** [review]
NPM version freeze related changes

**Review by kedarchandrayan — APPROVED** [review]
Looks good for merge.

**Review by deepesh-kn — APPROVED** [review]
LGTM

**Review by sunilkhedar — APPROVED** [review]
LGTM

**Pull request merged**